File: 1400/-1497.c
#include <stdio.h>
int main() {
    int num;
    int arr[101], max_arr[50];
    for (int i=0;i<num;i++)
        scanf("%d", &arr[i]);
    for (int i=0;i<num;i++){
        if (i%2==1)
            continue;
        else {
            if (arr[i] > arr[i+1])
                max_arr[i]=arr[i];
        }
    }
}

File: 1400/1497.py
num=int(input())
a=input()
arr=[]
data=[]
for i in range(num):
    arr.append(int(a.split(" ")[i]))
for i in range(num):
    if i%2:
        continue
    if arr[i] > arr[i+1]:
        data.append(arr[i])
    else:
        data.append(arr[i+1])
for i in data:
    print(i, end=" ")
